### PR TITLE
[PLATFORM-968] Create empty product

### PR DESCRIPTION
**Pull request by tumppi**
Add logic for POSTing an empty product when we have selected type from `ProductTypeChooser`.

Current backend will reject the call but it should start working after https://github.com/streamr-dev/engine-and-editor/pull/628 is merged.

### CODE REVIEW

File: app/src/marketplace/components/ProductTypeChooser/index.jsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ const ProductTypeChooser = ({ onSelect }: Props) => (
                     <button
                         type="button"
                         className={cx('btn', 'btn-special', styles.button)}
-                        onClick={() => onSelect('standard')}
+                        onClick={() => onSelect('NORMAL')}
```
Comment (juhah) [inline app/src/marketplace/components/ProductTypeChooser/index.jsx]: I can't remember if it was spec to leave out type for regular products or not?

File: app/src/marketplace/containers/CreateProductPage/index.jsx [hunk 2]
```diff
@@ -8,21 +8,28 @@ import { push } from 'connected-react-router'
...
+                id: product.id,
+            })))
+        } catch (err) {
+            console.error('Could not create an empty product', err)
+        }
     }, [dispatch])
 
+    const onTypeSelect = useCallback((type: ProductType) => {
+        createProduct(type)
+    }, [])
```
Comment (juhah) [inline app/src/marketplace/containers/CreateProductPage/index.jsx]: `createProduct ` should be in the dependency list.

File: app/src/marketplace/components/ProductTypeChooser/index.jsx (hunk 1 above)
Comment (tumppi) [reply app/src/marketplace/components/ProductTypeChooser/index.jsx]: We can leave it out in the service if needed. I think the backend code will default to `type = normal` if it is left out. I think it's good to be explicit if backend supports this.

Comment (juhah) [reply app/src/marketplace/components/ProductTypeChooser/index.jsx]: Yeah, let's go with this 👍

**Pull request merged**